src/OpenTelemetry.AutoInstrumentation/Configurations/EnvironmentConfigurationMetricHelper.cs

```diff
@@ -103,8 +104,15 @@ public static MeterProviderBuilder AddAspNetCoreInstrumentation(MeterProviderBui
         [MethodImpl(MethodImplOptions.NoInlining)]
         public static MeterProviderBuilder AddHttpClientInstrumentation(MeterProviderBuilder builder, LazyInstrumentationLoader lazyInstrumentationLoader)
         {
-            DelayedInitialization.Metrics.AddHttpClient(lazyInstrumentationLoader);
+#if NET6_0_OR_GREATER
+            if (DotNetVersionHelper.DotNetMajorVersion >= 8)
+            {
+                return builder.AddMeter("System.Net.Http")
+                    .AddMeter("System.Net.NameResolution");
```

Comment:
```suggestion
                return builder
                    .AddMeter("System.Net.Http")
                    .AddMeter("System.Net.NameResolution");
```

Reply:
Part of 7eeae793ffcf06bb360815057fccb62b67885da